src/datasim_ui/functions.cljs

```diff
@@ -62,30 +62,3 @@
         (.append js/document.body
                  link)))
     (.click link)))
-
-(defn import-url
-  [e k]
-  (ps-event e)
-  (dispatch [:dialog/open
-             {:title "Import from URL"
-              :form  {:url {:type  :text
-                            :label "URL"
-                            :text  ""}}
-              :save  (fn []
-                       (go (let [response
-                                 (<! (http/get (str api "/download-url")
-                                               {:with-credentials? false
-                                                :headers           {"Access-Control-Allow-Origin" "*"}
-                                                :query-params      {"url" (js/encodeURIComponent @(subscribe [:dialog.form/text :url]))}}))]
```

Comment:
Similar to my comment on the server side, is this subscription + `db` key still used or is it now unused?

Reply:
Those are somewhat generic subs/handlers used for lots of form elements, not specific to this URL. Also this was badly designed and the HANDLER for this data should have been in handlers but was instead inline in the component (the exact thing being deleted here).